src/framework/uicomponents/qml/MuseScore/UiComponents/StyledMenu.qml

```diff
@@ -136,8 +136,14 @@ StyledPopupView {
             }
         }
 
-        var itemHeight = (view.contentHeight - view.spacing * (model.length - 1)) / model.length
-        root.contentHeight = view.contentHeight - sepCount * (itemHeight - prv.separatorHeight) +
+        var itemHeight = 0
+        for(var child in view.contentItem.children) {
```

Comment:
No space between `for` and `(` in qml files, only in cpp ones? Looks like an inconsistent code style to me...